src/main/java/com/example/bookstore/config/SecurityConfig.java

```diff
@@ -31,12 +36,21 @@ public SecurityFilterChain web(HttpSecurity http) throws Exception {
                 .cors(AbstractHttpConfigurer::disable)
                 .csrf(AbstractHttpConfigurer::disable)
                 .authorizeHttpRequests((authorize) -> authorize
-                        .requestMatchers("/auth/**", "/error",
+                        .requestMatchers("/auth/**", "/error","/auth/login",

```

Comment:
strange to see "/auth/**" and "/auth/login" together, first one should cover the second